tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -1035,6 +1035,44 @@ public async Task AcquireTokenByRefreshTokenTestAsync()
             }
         }
 
+        [TestMethod]
```

Comment:
This test isn't an integration test, since it doesn't hit AAD. Moved to unit tests.